### DISCUSSION TITLE
Regarding  GSoC 2025

### DISCUSSION THREAD

**Discussion by codinglizard123** [oppia/oppia-android#5586] [GSoC 2025] [answered]
I’m interested in finding out whether the Oppia Android project will be part of Google Summer of Code (GSoC) 2025. Could you provide more details about this?

**Comment by adhiamboperes** [comment, marked as answer]
@codinglizard123 Yes, Oppia Android plans to participate in GSoC 2025. The list of projects will be shared in early 2025, aligning with the GSoC program timelines.

In the meantime, feel free to explore the [projects](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2024#android-team-1) we worked on this year.

We also encourage you to start contributing to the repository to get acquainted with the codebase and our code submission process.
